<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>19</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>13</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>File caricato:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="filePathText">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>40</y>
       <width>611</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QToolButton" name="selectInputButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleziona il file</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="settingsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>159</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Impostazioni</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Istanza</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="instanceComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Simmetrica</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Asimettrica</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Work Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="workModeComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>75</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Mode 0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mode 2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="workBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>279</y>
      <width>621</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Work</string>
    </property>
    <widget class="QToolButton" name="startWorkButton">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Avvia</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
